platform_versioned_docs/version-24.1/cloud/changelog.mdx

```diff
@@ -5,13 +5,36 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
-### v24.2.0_cycle23 Cloud - 2 September 2024
+### v24.2.0_cycle24 25 September 2024
+
+- Feature: Azure service principal credentials
+- Feature: Data Studios: Add direct Data Explorer browse link to mounted data items
+- Feature: Data Studios: Make StudioDetailsPage tabs routable
+- Feature: define cache configuration once, only override changes
+- Improvement: Data Explorer: Highlight invalid custom datalinks
+- Improvement: Disable AWS and Google Batch spot instance auto retry
+- Improvement: Data explorer: Remove folder validation
+- Improvement: Update 204 pipeline schema response description
+- Improvement: Patch workflow revision
+- Improvement: Render Parameters UI correctly with new schemas based on 2020-12 draft
+- Improvement: Upgrade to Angular 16
+- Improvement: Pipeline launch form: Delegate form creation to a form builder service
+- Improvement: Add Posthog integration
```

Comment:
@drewdipalma any additional comment on posthog wording before we go live with the docs?

Reply:
Per Drew, removing the Posthog line.